apps/main/src/lend/components/PageLoanManage/LoanRepay/index.tsx

```diff
@@ -82,6 +82,12 @@ const LoanRepay = ({
   const { decimals: borrowedTokenDecimals } = borrowed_token ?? {}
   const { expectedBorrowed } = detailInfoLeverage ?? {}
 
+  const { data: loanExists } = useLoanExists({
+    chainId: rChainId,
+    marketId: market?.id,
+    userAddress: signerAddress as Address,
```

Comment:
this is super annoying. I think we can hack the type over here
https://github.com/curvefi/curve-frontend/blob/55b5f2e6e94b6bc308ece132a5193146482afe5d/packages/curve-ui-kit/src/features/connect-wallet/lib/types.ts#L20-L21